.github/actions/check-pr/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "check-pr",
```

Comment:
[q] Could this be in a separate repo and just be released to NPM (and then called with `npx` or as a github action step) to avoid cluttering the SDK repo?

Reply:
It could, but in my opinion this would be major overkill...

Reply:
Yes, they are, example => https://github.com/SAP/cloud-sdk-js/pull/2776. I ❤️ GH.

Reply:
I can add this publishing idea to the ticket. But I really would like to avoid additional complexity. Commiting node_modules is ugly, but not complex. Making changes in a separate repo and publishing is nicer, but much more complex.